{"name": "test_get_token[caseinfo2]", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestApi' object has no attribute 'logging'", "trace": "self = <test_case.gzh_manage.Test_gzh.TestApi object at 0x00000000049FB4A8>\ncaseinfo = {'name': '获取鉴权码', 'request': {'data': {'grant_type': 'client_credential', 'secret': '6a3af6156f86205146d1abcadc13dd8d'...': 'get', 'url': 'https://api.weixin.qq.com/cgi-bin/token'}, 'validate': {'assert_code': 200, 'assert_str': 'errcode'}}\n\n    @pytest.mark.parametrize('caseinfo',read_yaml('test_case/gzh_manage/get_token.yaml'))\n    @pytest.mark.run(order=1)\n    def test_get_token(self,caseinfo):\n        method = caseinfo['request'][\"method\"]\n        url = caseinfo['request'][\"url\"]\n        data = caseinfo['request'][\"data\"]\n        res=RequestsUtil.send_requests(method,url,data)\n>       self.logging.info(\"请求地址:{}\".format(url))\nE       AttributeError: 'TestApi' object has no attribute 'logging'\n\ntest_case\\gzh_manage\\Test_gzh.py:21: AttributeError"}, "parameters": [{"name": "caseinfo", "value": "{'name': '获取鉴权码', 'request': {'method': 'get', 'url': 'https://api.weixin.qq.com/cgi-bin/token', 'data': {'grant_type': 'client_credential', 'secret': '6a3af6156f86205146d1abcadc13dd8d'}}, 'validate': {'assert_str': 'errcode', 'assert_code': 200}}"}], "start": 1653271600889, "stop": 1653271600948, "uuid": "18f6c3d0-9e02-4f17-87e3-3939e9672d2b", "historyId": "79169f70909ad2dba76774288528e3fa", "testCaseId": "89a275626639f8a5e9d51334136f29e5", "fullName": "test_case.gzh_manage.Test_gzh.TestApi#test_get_token", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_case.gzh_manage"}, {"name": "suite", "value": "Test_gzh"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "PC-20180915DYDK"}, {"name": "thread", "value": "88956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.gzh_manage.Test_gzh"}]}